openssl/src/ssl/mod.rs

```diff
@@ -342,6 +433,36 @@ impl SslContext {
         };
         SslContextOptions::from_bits(ret).unwrap()
     }
+
+    /// Set the protocols to be used during Next Protocol Negotiation (the protocols that
+    /// the client supports).
```

Comment:
Could you add a note to the docs here mentioning that this method needs the npn feature?